addons/account/tests/test_tax_report.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.account.tests.account_test_savepoint import AccountingSavepointCase
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TaxReportTest(AccountingSavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TaxReportTest, cls).setUpClass()
+        cls.test_country_1 = cls.env['res.country'].create({
+            'name': "L'Île de la Mouche",
+            'code': 'YY',
+        })
+
+        cls.test_country_2 = cls.env['res.country'].create({
+            'name': "La Lande aux Alouettes",
+            'code': 'ZZ',
+        })
+        cls.test_country_3 = cls.env['res.country'].create({
+            'name': "Le Pays Joyeux des Enfants Heureux",
```

Comment:
as funny as it is, I wouldn't name those countries in french so that, non-french speakers wouldn't ask themselves if their names are meaningful for the sake of the test case